bot/models/levels.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+from .model import Model
+
+
+class Levels(Model):
+    rank: Optional[int]
+    guild_id: int
+    user_id: int
+    old_total_xp: Optional[int]
+    total_xp: int
```

Comment:
Like before, all tables should have a unique id column

```suggestion
class Levels(Model):
    id: int
    guild_id: int
    user_id: int
    total_xp: int
```